src/librustdoc/html/static/storage.js

```diff
@@ -84,7 +84,7 @@ function onEachLazy(lazyArray, func, reversed) {
 }
 
 // eslint-disable-next-line no-unused-vars
```

Comment:
Why do we need to disable the no-unused-vars lint on this?

Reply:
Because it's not used inside this file.